alvr/dashboard/src/dashboard/components/setup_wizard.rs

```diff
@@ -113,7 +113,7 @@ Make sure you have at least one output audio device.",
                 ui,
                 "Software requirements",
                 if cfg!(windows) {
-                    r"To stream the headset microphone on Windows you need to install VB-Cable or Voicemeeter."
+                    r"To stream the headset microphone on Windows you need to install Virtual Audio Cable, VB-Cable, or Voicemeeter."
```

Comment:
Given Zmerp's input, you should probably either fully remove this particular change or place it at the end of the list.